UPGRADE.md

```diff
@@ -10,9 +10,19 @@ The `autocompile` option is now the default option for the behavior of `\gregori
 
 ### Gregorio and Grana Padano fonts
 
-The gregorio and granapadano fonts, as well as their "-op" Dominican variants, will now be available as separate downloads.  If you were using these fonts, either download and install them or switch to greciliae, which continues to be distributed along with Gregorio by default.
+The Gregorio and Grana Padano fonts, as well as their "-op" Dominican variants, will now be available as a separate download.  If you were using these fonts, either download `supp_fonts-5_0_0.zip` and install them or switch to Greciliae, which continues to be distributed (with its "-op" Dominican variants) along with Gregorio by default.
```

Comment:
Gregorio was the original font that came with Gregorio.  Greciliae was adapted from the Caeciliae font and pretty much everyone agrees it looks better than Gregorio.  History being what it is, I think we should just leave these names alone.  The Parmesan rename was necessary because of a collision with the "real" Parmesan font from LilyPond.